sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -201,6 +201,30 @@ gnrc_pktsnip_t *gnrc_ndp_opt_pi_build(const ipv6_addr_t *prefix,
     return pkt;
 }
 
+gnrc_pktsnip_t *gnrc_ndp_opt_ri_build(const ipv6_addr_t *prefix,
+                                      uint8_t prefix_len,
+                                      uint32_t route_ltime,
+                                      uint8_t flags, gnrc_pktsnip_t *next)
```

Comment:
I think this must be made optional as well, otherwise I can't explain why it still isn't fitting all boards.

Reply:
Or maybe put it in a different PR, as it isn't used here anyways?

Reply:
huh that's weird - why isn't this discarded with `-ffunction-sections`?

Reply:
I'm not even sure this function is the problem, but the added NOPs to the switch-cases seem to be to small to cause a 52 byte overflow as we see on `tests/gnrc_sixlowpan_frag_minfwd` for `stm32mp157c-dk2` and this is the only other larger code section added.